app/views/admin/media/_modal.html.slim

```diff
@@ -1,7 +1,6 @@
 .modal.media-modal
   div.modal-dialog
     div.modal-content
-      // Content to be filled in by JS via AJAX.
```

Comment:
Did this get removed instead of the other one? This one still makes sense I think...